package.json

```diff
@@ -108,15 +108,20 @@
         "vscode-azureextensiondev": "^0.3.1"
     },
     "dependencies": {
+        "@octokit/rest": "^17.11.0",
         "azure-arm-resource": "7.3.0",
```

Comment:
Lots of new packages being added? Are all of these used? I do not see them being imported anywhere

Reply:
Earlier I was using Octokit to push files into GitHub repository, this was using globby objects and some functions from the package fs-extra, but since now simple-git is being used to push the files, these packages are rendered useless. 
Similarly, earlier we were taking the repo name as input from users, that's why package github-reserved-names was installed.
I'll delete all these packages.